src/app/modals/about-modal.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="about-modal-pf">
+  <div class="modal-header">
+    <button type="button" class="close" (click)="close($event)" aria-hidden="true">
+      <span class="pficon pficon-close"></span>
+    </button>
+  </div>
+  <div class="modal-body">
+    <h1 *ngIf="config.title">{{config.title}}</h1>
+    <div *ngIf="config.productInfo && config.productInfo.length > 0" class="product-versions-pf">
+      <ul class="list-unstyled">
+        <li *ngFor="let info of config.productInfo">
+          <strong>{{info.name}}</strong> {{info.value}}</li>
+      </ul>
+    </div>
+    <div class="product-versions-pf">
+      <ng-template *ngIf="template" [ngTemplateOutlet]="template">
+      </ng-template>
+    </div>
+    <div *ngIf="config.additionalInfo" class="product-versions-pf">{{config.additionalInfo}}</div>
+    <div *ngIf="config.copyright" class="trademark-pf">{{config.copyright}}</div>
+  </div>
+  <div class="modal-footer">
+    <img *ngIf="config.imgSrc" [src]="config.imgSrc" alt="{{config.imgAlt}}" />
+  </div>
+</div>
```

Comment:
Need a return char.

FYI, I added a TS linter and AOT compilation to patternfly-ng. The build should show these issues automatically, but I may need to tweak some settings?